src/model/model.ts

```diff
@@ -202,12 +203,7 @@ export class Model {
   }
 
   // getFilteredPolicy gets rules based on field filters from a policy.
-  public getFilteredPolicy(
```

Comment:
I already fixed this in: https://github.com/casbin/node-casbin/commit/7e78cbaa9b4461cb7f241743fa83c4afcd107be8. Please remove it.